lib/ibm_watson/visual_recognition_v4.rb

```diff
@@ -29,6 +29,8 @@ module IBMWatson
   # The Visual Recognition V4 service.
   class VisualRecognitionV4 < IBMCloudSdkCore::BaseService
     include Concurrent::Async
+    DEFAULT_SERVICE_NAME = "watson_vision_combined"
```

Comment:
this should be `visual_recognition`, I have added the missing handwritten changes for `vis rec` to the wiki.